grass7/imagery/i.modis/i.modis.download/i.modis.download.py

```diff
@@ -255,7 +261,7 @@ def main():
         modisOgg.connect()
         if modisOgg.nconnection <= 20:
             # download tha tiles
-            grass.message(_("Downloading MODIS product <%s>..." % produ))
+            grass.message(_("Downloading MODIS product <%s> (%s)..." % (produ, prod['prod'])))
```

Comment:
general question: what is the difference between `%s % ()` and the .format() way?

Reply:
I think format should be preferable. Reading [this](https://stackoverflow.com/questions/5082452/string-formatting-vs-format-vs-f-string-literal) format seems slower...

Reply:
ok, changed by Anika :)